src/Symfony/Component/Translation/Bridge/Loco/Tests/LocoProviderFactoryTest.php

```diff
@@ -34,6 +34,11 @@ public static function createProvider(): iterable
             'loco://localise.biz',
             'loco://API_KEY@default',
         ];
+
+        yield [
+            'loco://localise.biz',
```

Comment:
Status should be kept in toString, it is no sensitive information

Reply:
This would break BC as the now-static `status=translated,blank-translation` would then be appended to the endpoint in `__toString`. Would this be a problem?

Reply:
No problem, as it would only be present if proposed by the user

Reply:
ok, done